### PR TITLE
adjust timezone config description

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

Since December 23, 2021, the TiDB DM documentation in the master branch has been merged to TiDB documentation by [#8042](https://github.com/pingcap/docs-cn/pull/8042) and [#7317](https://github.com/pingcap/docs/pull/7317). To create a PR for TiDB DM versions later than v5.3, either go to TiDB [English documentation repository](https://github.com/pingcap/docs) or [Chinese documentation repository](https://github.com/pingcap/docs-cn).

- [x] v5.3 (TiDB DM 5.3 versions)
- [ ] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): Cherry-pick from https://github.com/pingcap/docs-cn/pull/8159

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@glorv Please sign the [Contributor License Agreement](https://cla-assistant.io/pingcap/docs-dm?pullRequest=885). Thanks.

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ is-sharding: true               # `is-sharding` has been deprecated since v2.0.0
 shard-mode: "pessimistic"       # The shard merge mode. Optional modes are ""/"pessimistic"/"optimistic". The "" mode is used by default which means sharding DDL merge is disabled. If the task is a shard merge task, set it to the "pessimistic" mode. 
                                 # After understanding the principles and restrictions of the "optimistic" mode, you can set it to the "optimistic" mode.
 meta-schema: "dm_meta"          # The downstream database that stores the `meta` information.
-timezone: "Asia/Shanghai"       # The timezone.
+timezone: "Asia/Shanghai"       # The timezone use in SQL Session. Buy default, DM will use the global timezone setting in target cluster, and can ensure the correctness automatically. Manually set to other timezone won't affect the correctness, so I most can, you needn't change it.
```
Comment (shichun-0415) [inline en/task-configuration-file-full.md:30]:
```suggestion
timezone: "Asia/Shanghai"       # The timezone used in SQL Session. By default, DM uses the global timezone setting in the target cluster, which ensures the correctness automatically. Customized timezone does not affect data migration but is unnecessary.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: en/task-configuration-file-full.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ is-sharding: true               # `is-sharding` has been deprecated since v2.0.0
 shard-mode: "pessimistic"       # The shard merge mode. Optional modes are ""/"pessimistic"/"optimistic". The "" mode is used by default which means sharding DDL merge is disabled. If the task is a shard merge task, set it to the "pessimistic" mode. 
                                 # After understanding the principles and restrictions of the "optimistic" mode, you can set it to the "optimistic" mode.
 meta-schema: "dm_meta"          # The downstream database that stores the `meta` information.
-timezone: "Asia/Shanghai"       # The timezone.
+timezone: "Asia/Shanghai"       # The timezone used in SQL Session. By default, DM uses the global timezone setting in the target cluster, which ensures the correctness automatically. Customized timezone does not affect data migration but is unnecessary.
```
Comment (shichun-0415) [inline en/task-configuration-file-full.md:30]:
```suggestion
timezone: "Asia/Shanghai"       # The timezone used in SQL Session. By default, DM uses the global timezone setting in the target cluster, which ensures the correctness automatically. A customized timezone does not affect data migration but is unnecessary.
```
```suggestion
timezone: "Asia/Shanghai"       # The timezone used in SQL Session. By default, DM uses the global timezone setting in the target cluster, which ensures the correctness automatically. Customized timezone does not affect data migration but is unnecessary.
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**